src/components/fullscreen/fullscreen.js

```diff
@@ -64,6 +64,7 @@ class FullscreenControl extends BaseComponent {
       typeof document.mozFullScreenElement !== 'undefined' && Boolean(document.mozFullScreenElement) ||
       typeof document.msFullscreenElement !== 'undefined' && Boolean(document.msFullscreenElement);
 
+    !isFullscreen ? this.player.notifyExitFullscreen() : this.player.notifyEnterFullscreen();
```

Comment:
change to positive condition
`isFullscreen ?  this.player.notifyEnterFullscreen() :  this.player.notifyExitFullscreen();`
is more intuitive.